apps/davi/src/components/ProposalStatus/ProposalStatus.tsx

```diff
@@ -96,13 +99,19 @@ export const ProposalStatus: React.FC<ProposalStatusProps> = ({
           </DetailText>
         )}
         {status ? (
-          <ProposalStatusDetail
-            data-testid="proposal-state"
-            statusDetail={status}
+          <ExternalLink
+            href={executeTxLink ?? null}
+            disableLink={!executeTxLink}
+            hideIcon
           >
-            {' '}
-            {status}
-          </ProposalStatusDetail>
+            <ProposalStatusDetail
+              data-testid="proposal-state"
+              statusDetail={status}
+            >
+              {' '}

```

Comment:
🧹  {'  '}

Reply:
Interesting!

In the case of executionTxHash, we check for falsy values since it can be either null (if we're in a subgraph-supported network) or undefined (if we're in a non-subgraph-supported network).

In the components, the "!" seems, in my opinion, to improve readability. Like: "if not hideIcon, then render this component", but I guess you can make the same argument for "if hideIcon is false, then show this".

---

I'll remove the `{' '}` in this issue(#196), since I'll be working with that component. Good catch.